### DISCUSSION TITLE
setup.php does not offer login

### DISCUSSION THREAD

**Discussion by xBlackfire89** [postfixadmin/postfixadmin#827] [General]
Hello,

im using PHP 8.3 on Debian 12 and use the Master Branch of Postfixadmin, i have installed everything according to a tutorial and also have read the install.txt etc.

When opening the setup.php the Database structure is added but it says:
 "You need to have a setup_pasword hash configured in a config.local.php file  "
No matter how often i generate the password hash and add it to my config.local.php it does not understand that there is a password therefore it does not offer the login.
i wanted to try to edit the setup.php in public folder, but even deleting it didnt show any reaction to calling the setup.php (meaning it still shows the page and works as before)... what am i missing?
Greetings a frustrated Server Admin :'D

**Comment by DavidGoodwin** [comment]
Hi - does this poorly produced video help? That's me running what's in 'master' with before a setup password is generated, and once one is added in to the config, creating a global administrator (some-admin@example.com).


![postfixadmin-master-setup-password-etc](https://github.com/postfixadmin/postfixadmin/assets/203929/8a6a881a-26ec-4642-82b3-3a25878c2a17)

**Reply by xBlackfire89** [reply]
Hello and thanks for your reply :) especially after the "refresh" when the login box would appear, it does not appear for me.

**Reply by DavidGoodwin** [reply]
Hm, weird. The file I'm editing is `config.local.php` in my postfixadmin source directory.

If the login box doesn't appear it's possible you're not editing the right config file - sometimes they're in e.g. /etc/postfixadmin ?

**Reply by xBlackfire89** [reply]
i run a file search above everything i have a folder /usr/share/postfixadmin which has another config file that is a link to /etc/postfixadmin... so i edited the wrong one? or is there from my old try still an installation of the debian package interfering?

**Reply by xBlackfire89** [reply]
i edited the other config and it worked... now im still confused but it did work... so im wondering now wheter i did something wrong or why the config is there?

**Reply by DavidGoodwin** [reply]
I can only suggest double checking your apache (?) config and make sure the Postfixadmin you are executing/using is what you expect !
